components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/service/notification/NotificationTemplateManager.java

```diff
@@ -42,6 +42,24 @@ default NotificationTemplate getNotificationTemplate(String notificationChannel,
         return null;
     }
 
+    /**
+     * Return the notification template from the tenant registry which matches the given channel and template name.
+     *
+     * @param notificationChannel Notification Channel Name (Eg: SMS or EMAIL)
```

Comment:
Add fullstops at the end

Reply:
Addressed with https://github.com/wso2-extensions/identity-governance/pull/811/commits/58587abf4c12a0b457e48a41093beb5709daa915